code/src/App.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react'
```

Comment:
You are very consistent with your code Ingela and that is really great! 
For example you always use single quote and no extra semi colons so that is very nice :)